docs/blog/2019-08-30-speed-up-your-time-to-interactive-by-delaying-third-party-scripts/index.md

```diff
@@ -68,4 +68,4 @@ There is currently a [PR](https://github.com/benjaminhoffman/gatsby-plugin-segme
 
 ## In closing
 
-If you work on a laptop or desktop with a good internet connection, the Time to Interactive is not so noticable as "time before you see something on your screen", but it can have a big impact on your performance rating and interactivity, and because of that, your SEO. You should be extra careful when adding third party scripts to your page, and the best way to do that is to run Google Lighthouse. If you notice your Gatsby site has a bad performance score, make sure to check the Time to Interactive value. If it's very high, try loading scripts only after a user has interacted with your website.
+If you work on a laptop or desktop with a good internet connection, the Time to Interactive is not so noticeable as "time before you see something on your screen", but it can have a big impact on your performance rating and interactivity, and because of that, your SEO. You should be extra careful when adding third party scripts to your page, and the best way to do that is to run Google Lighthouse. If you notice your Gatsby site has a bad performance score, make sure to check the Time to Interactive value. If it's very high, try loading scripts only after a user has interacted with your website.
```

Comment:
**FYI:** The word **noticable** is changed to **noticeable**

Unfortunately, it is not noticeable in the diff :rofl: